index.d.ts

```diff
@@ -136,11 +136,11 @@ declare namespace pc {
         
         update: (dt: number) => boolean;
         
-        on: (
-            name: string,
-            callback: callbacks.HandleEvent,
```

Comment:
I was unable to find information on this type here `callbacks.HandleEvent` let me know if it more appropriate to use that over `() => void`.

Reply:
No, I think `() => void` is fine here. 😄